setup.py

```diff
@@ -1,18 +1,18 @@
 from setuptools import find_packages, setup
 
-from aiolookin import __version__ as version
-
 setup(
     name="aiolookin",
-    version=version,
+    version="0.1.0",
```

Comment:
`pip3 install --upgrade .` failed because `setup.py` tries to import `aiolookin` which requires `aiohttp` so I set this here and reconfigured `bump2version` to be able to change it.